simulators/nav/src/components/NavSimulator.vue

```diff
@@ -437,29 +443,29 @@ export default class NavSimulator extends Vue {
 
     /* Set up publishing LCMs */
     this.intervalLcmPublish = window.setInterval(() => {
-      this.publish('/auton', { type: 'AutonState', is_auton: this.autonOn });
+      this.publish('/auton', { type: 'AutonState', is_auton: this.autonOn }, false);
```

Comment:
why is this set to false?

Reply:
This is just a selector for the Enable LCM function. True if we want to always publish the channel regardless of the state of Enable LCM